src/data_storage/connection.py

```diff
@@ -0,0 +1,12 @@
+from conf import settings
+
+from .data_source import DataDriver
+from .dynamo_db.data_source import DynamoDBDriver
+from .sqlalchemy.data_source import RDSDriver
+
+
+def get_data_source() -> DataDriver:
+    if settings.DATA_SOURCE == "dynamodb":
+        return DynamoDBDriver()
+    elif settings.DATA_SOURCE == "rds":
+        return RDSDriver()
```

Comment:
I'd keep Driver initializing logic here
```suggestion

def get_data_source() -> DataDriver:
    if settings.DATA_SOURCE == "dynamodb":
        return DynamoDBDriver(**settings.data_source_settings)
    elif settings.DATA_SOURCE == "rds":
        return RDSDriver(**settings.data_source_settings)
```

Reply:
But here's a duplication of `if settings.DATA_SOURCE == "dynamodb"`.

Reply:
So, we can make do here something like: 

``` python
DATA_SOURCES = {
    "dynamodb": (DynamoDBDriver, DynamoDBSettings),
    "rds": (RDSDriver, RDSSettings),
}

def get_data_source() -> DataDriver:
    data_source = DATA_SOURCES.get(settings.data_sourse)
    if data_source is None:
        raise ValueError("Unknown driver name")
    driver_cls, settings_cls = data_source
    data_source_settings = settings_cls()
    driver = driver_cls(**data_source_settings)
    return driver
```

And we don't need `data_source_settings` property in this case.

Reply:
Make sense